### DISCUSSION TITLE
Is it possible to view Rbacassessmentreports in graphana

### DISCUSSION THREAD

**Discussion by travis-jorge** [aquasecurity/trivy-operator#1899] [Help & Support]
I have a dashboard that I can view the CVE info 
![image](https://github.com/aquasecurity/trivy-operator/assets/80355769/b07bbc8d-4c9b-4e96-9983-26023999c01b)
Is it possible to visualize the other report info like the rbac assessment reports

**Comment by chen-keinan** [comment]
@travis-jorge it is supported in term of metric, regarding grafana dashboard , you probably have to configure it, or use the [default](https://grafana.com/grafana/dashboards/17813-trivy-operator-dashboard/)
